src/lib/sentry.ts

```diff
@@ -1,8 +1,32 @@
+/**
+ * Importing these separately from `platform/detection` and `lib/app-info` to
+ * avoid future conflicts and/or circular deps
+ */
+
+import {Platform} from 'react-native'
+import app from 'react-native-version-number'
+import * as info from 'expo-updates'
 import {init} from 'sentry-expo'
 
+/**
+ * Matches the build profiles from `eas.json`
+ */
+const buildProfile = (info.channel || 'development') as
+  | 'development'
+  | 'preview'
+  | 'production'
+
+/**
+ * Examples:
+ * - `ios@1.57.0(3)`
+ * - `android@1.57.0(46)`
+ */
+const appVersion = `${Platform.OS}@${app.appVersion}(${app.buildVersion})`
```

Comment:
I also did this here so that this always stays constant in case we ever change how it displays in the app (Settings page shows this)